### ISSUE TITLE
Include errors while parsing (Boost) unit test code 

### ISSUE THREAD

**Issue by shishirmisra** [SonarOpenCommunity/sonar-cxx#285]
Hi.
   This is related to the second issue I noted in https://github.com/wenns/sonar-cxx/issues/276. The sonar output shows a set of errors for the main unit test CPP file. Basically, the test file includes the main header file present under the sonar.sources directory, and sonar-runner complains about missing headers from that main header -

14:53:52.918 WARN  - [Z:\src\lib\BlobStorage\BlobStorage_lib\BlobStorage.h:19]: cannot find the sources for '#include <boost/uuid/uuid.hpp>'
14:53:52.918 WARN  - [Z:\src\lib\BlobStorage\BlobStorage_lib\BlobStorage.h:20]: cannot find the sources for '#include <boost/uuid/uuid_generators.hpp>'

Similar errors are also seen for BlobStore_test.h header. This leads to these syntax errors during parsing -

14:53:53.086 WARN  - [Z:\src\lib\BlobStorage\BlobStorage_test\BlobStore_test.cpp:42]: syntax error, skip 'BOOST_FIXTURE_TEST_SUITE'

Basically, it looks like sonar-runner can't use sonar.cxx.includeDirectories while parsing code under sonar.tests. The same include paths definitely work while parsing code under sonar.sources. Should test files be parsed just like source files? If yes, then shouldn't the sonar.cxx.includeDirectories directive be sufficient? Or is another directive needed?
Regards,
Shishir
